appcues/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,9 @@
 <resources>
     <!-- SDK -->
     <string name="appcues_skippable_trait_dismiss">dismiss</string>
+    <string name="appcues_preview_embed_message">Please navigate to the screen with frame %1$s to preview %2$s.</string>
+    <string name="appcues_preview_flow_not_found">Flow not found.</string>
+    <string name="appcues_preview_preview_failed">Mobile flow preview failed.</string>
```

Comment:
using only one "generic" error message instead of two, or should I mirror IOS, maybe I didnt get the difference [here](https://github.com/appcues/appcues-ios-sdk/blob/b96359ca9c02e6f590bbf2cc244a9c0ccd448f9d/Sources/AppcuesKit/Presentation/DeepLinkHandler.swift#L141).

Reply:
"Flow not found" would be the 404 case - invalid id
"Mobile flow preview failed" - the network loading was OK, but this case covers anything that causes failed render like invalid element target

the missing one is
"Error loading mobile flow preview." - this would be a network failure for any reason other than 404 - like invalid JSON structure or some other connection issue